platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/command/AddApplicationCommandTest.java

```diff
@@ -41,7 +41,7 @@
     private static final String APPLICATION_SERVICE_NOT_FOUND =
             "ApplicationService not found";
 
-    private Logger logger = LoggerFactory.getLogger(AddApplicationCommand.class);
+    private Logger LOGGER = LoggerFactory.getLogger(AddApplicationCommand.class);
```

Comment:
We're starting to get out of scope here, but it doesn't look like this LOGGER is being used in the class